utils/retrieve-state/src/main.rs

```diff
@@ -79,14 +83,26 @@ struct Opts {
     use_gzip: bool,
```

Comment:
I suggest we just do this by default rather than have a flag.

Reply:
Done